admin_guide/sysctls.adoc

```diff
@@ -102,14 +103,14 @@ containers, resource shortage, or complete breakage of a node.
 ====
 
 [[enabling-unsafe-sysctls]]
-== Enabling Unsafe Sysctls
+== Enabling unsafe sysctls
 
-With the warning above in mind, the cluster administrator can allow certain
-unsafe sysctls for very special situations, e.g., high-performance or real-time
+With the previous warning in mind, the cluster administrator can allow certain
+unsafe sysctls for very special situations such as high-performance or real-time
```

Comment:
@kalexand-rh Should this be more explicit to point to the previous warning? 
```
Enabling unsafe sysctls

The use of unsafe sysctls is at-your-own-risk. See the xref:linktext[previous warning].

The cluster administrator can allow...
```

Reply:
You're right that it could be clearer, but I don't want to add an xref that I'll have to remove next month. I might move that paragraph into the previous section - if these were modules, that detail would belong in the concept instead of the task.